### ISSUE TITLE
BUG: special: autodifferentiation of division operator

### ISSUE THREAD

**Issue by mdhaber** [scipy/scipy#21665]
### Describe your issue.

In gh-21568, we found that the implementation of `dual &operator/=(const dual &other)` may not be quite right. I thought we should open this so it's not forgotten. 
@izaid can you fix this, and @steppi can review?
In any case, we should add a test and a reference so it doesn't come up again.

**Comment by mdhaber** [comment]
gh-21711 made changes, but I'll leave this open as a reminder that it could use a reference and test.

**Comment by ev-br** [comment]
Have bumped the milestone. Might be great to actually write a test.  Maybe we should even transfer this issue to `xsf`, where the dual math now lives.

**Comment by rgommers** [comment]
Removed the milestone, looks like this can be done at any time and shouldn't block a release.

**Comment by steppi** [comment]
I agree with @ev-br that this issue should be transferred to `xsf`. The tests are a more natural fit for there. Closing as not planned, and I'll post an issue over there.
